app/about_us/OurCrew.js

```diff
@@ -0,0 +1,62 @@
+import "./OurCrew.css";
+
+const crewMembers = [
```

Comment:
As an advanced tip, it is a good idea to move data definitions like this to their own separate file (and even folder), to separate _data_ from _logic_. You can, for example, store it as a JSON file.